### PR TITLE
Fix build errors in cgo_library rules at the top level package

### LINKED ISSUE #23: cgo_library build fails with `/bin/bash: line 0: cd: HOME not set`
Building the following cgo project fails with `/bin/bash: line 0: cd: HOME not set`

---

    $ ls
    bazel-bin  bazel-cgotest  bazel-genfiles  bazel-out  bazel-testlogs  BUILD  cgo.go  WORKSPACE

---
    cat BUILD 
    load("@io_bazel_rules_go//go:def.bzl", "go_prefix", "cgo_library")
    
    go_prefix("cgotest")
    
    cgo_library(
        name = "cgo",
        srcs = ["cgo.go"],
    )

---

    $ cat WORKSPACE 
    git_repository(
        name = "io_bazel_rules_go",
        remote = "https://github.com/bazelbuild/rules_go.git",
        tag = "0.0.2",
    )
    load("@io_bazel_rules_go//go:def.bzl", "go_repositories")
    go_repositories()

The build itself

    $ ~/bin/bazel build //:cgo --verbose_failures
    WARNING: Sandboxed execution is not supported on your system and thus hermeticity of actions cannot be guaranteed. See http://bazel.io/docs/bazel-user-manual.html#sandboxing for more information. You can turn off this warning via --ignore_unsupported_sandboxing.
    INFO: Found 1 target...
    ERROR: /vagrant/cgotest/BUILD:5:1: error executing shell command: 'export GOROOT=$(pwd)/external/golang_linux_amd64/go/bin/.. && export CC=$(cd $(dirname /usr/bin/gcc); pwd)/$(basename /usr/bin/gcc) && export CXX=$CC && objdir=$(pwd)/bazel-out/local-fastbuild/genf...' failed: bash failed: error executing command 
    (cd /home/vagrant/.cache/bazel/_bazel_vagrant/b4ea8fffb29363907600583bb5450a23/cgotest && \
    exec env - \
        CGO_LDFLAGS='' \
        GOARCH=amd64 \
        GOOS=linux \
    /bin/bash -c 'export GOROOT=$(pwd)/external/golang_linux_amd64/go/bin/.. && export CC=$(cd $(dirname /usr/bin/gcc); pwd)/$(basename /usr/bin/gcc) && export CXX=$CC && objdir=$(pwd)/bazel-out/local-fastbuild/genfiles/cgo.cgo.dir && mkdir -p $objdir && cd  && $GOROOT/bin/go tool cgo -objdir $objdir -- cgo.go && rm -f $objdir/_cgo_.o $objdir/_cgo_flags'): com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 1.
    /bin/bash: line 0: cd: HOME not set
    Target //:cgo failed to build
    INFO: Elapsed time: 0.463s, Critical Path: 0.08s

I think the cause is that `ctx.label.package` on line https://github.com/bazelbuild/rules_go/blob/master/go/def.bzl#L522 is an empty string. I tried figuring out what is supposed to initialize that variable, but I could not in the time I was willing to spend on it.

If I change `cgo_library` to `go_library` and comment out all cgo parts from `cgo.go`, then compilation succeeds.

I am using RHEL 7.2, OpenJDK 1.8 and current release of Bazel.

### PR DESCRIPTION
**Pull request by yugui**
Fixes #23

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by yugui** [comment]
test this please

**Comment by yugui** [comment]
Let me try again, Jenkins.

test this please

**Comment by damienmg** [comment]
LGTM
